packages/website/docs/api-2d.md

```diff
@@ -2236,6 +2236,146 @@ function Mouse(options?: {
 };
 ```
 
+### Ogmo.Project
+
+> Available since version: Unreleased
+
+```ts
+import { Ogmo, useNewComponent } from "@hex-engine/2d";
+import myProject from "./myProject.ogmo";
+
+useNewComponent(() => Ogmo.Project(myProject));
+```
+
+A Component that provides an interface for working with an [Ogmo Editor](https://ogmo-editor-3.github.io/) project.
+
+> Note: Several of the types in this overview are omitted. Rely on your IDE for definitions of these types.
+
+````ts
+function OgmoProject(
+  /**
+   * The imported *.ogmo file
+   *
+   * Available since version: Unreleased
+   */
+  projectData: any,
+
+  /**
+   * An object map of functions that will be used to
+   * construct entities in Ogmo levels, by name; for example:
+   *
+   * { player: (entData) => useChild(() => Player(entData)) }
+   *
+   * Available since version: Unreleased
+   */
+  entityFactories: {
+    [name: string]: (entityData: OgmoEntityData) => Entity;
+  } = {},
+
+  /**
+   * An optional function that will be called to construct entities for decals.
+   * The default implementation uses Ogmo.Decal.
+   *
+   * Available since version: Unreleased
+   */
+  decalFactory?: (decalData: OgmoDecalData) => Entity
```

Comment:
wouldn't we potentially want to treat decals differently similar to entities? i.e. do we not want factories here instead of a single function?

Reply:
Potentially, but there's not a clean separation for which decal is which like there is with entities. You can still treat decals differently within this function, though, using an `if` to check the decal data and then returning different entities.